src/main/java/org/vandeseer/easytable/TableDrawer.java

```diff
@@ -139,11 +155,15 @@ public void draw(Supplier<PDDocument> documentSupplier, Supplier<PDPage> pageSup
     
     protected PDPage determinePageToDraw(int index, PDDocument document, Supplier<PDPage> pageSupplier) {
```

Comment:
Is the returned value used at all? So either return the value and assign it where it is called (preferably) or just make the method `void`.

Reply:
The returned PDPage  is used by the draw(Supplier<PDDocument> documentSupplier, Supplier<PDPage> pageSupplier, float yOffset) method.

Reply:
Sorry, my bad. You are right. Still, there is a thing which I don't like about this method and which should be easy to fix: There is a side effect, namely setting the member variable `tableStartPage` within the method. At the same time this value is returned. Since it is called only once one could argue that it would make more sense to set the member variable after the method is called using its returned value.

(There is a 2nd side effect nonetheless though which is setting the member variable ` startTableInNewPage` – but the fewer side effects the better in my opinion.)

Reply:
Point taken. Have moved both the if conditions into the parent draw method.